src/app/util/BUILD.gn

```diff
@@ -52,7 +52,7 @@ source_set("types") {
   public_configs = [ "${chip_root}/src:includes" ]
 }
 
-# This source set also depends on data-model
+# This source set also depends on data-model and platform_config_header
```

Comment:
These comments are odd: we can see these dependencies in the `deps` bit.

How about we just drop this comment ... I am not quite clear how this comment helps me read the code.

Reply:
The original comment is not of my doing. I just adapted it so it remains correct for the block it comments.

I am a (BTW 100% self-funded, that is, not funded at all) outside developer, not an insider, and as such I can't grow into everything else that might be debatable in the proximity of this PR (such as if this comment should be kept in the first place). This case is a minor one of course, but it demonstrates the resource problem I slipped into with this PR quite well ;-)

Reply:
Git blame says the original comment is my doing...and now that I look at it it does not seem useful. I would rather delete it than keep extending it with more duplication of what `deps` say.

Reply:
So l remove both of them, the one at line 32, too?

Reply:
I think so. The test would be: do they help you understand the code better? If they do not, the seem like not useful.